include/filemenu.h

```diff
@@ -18,28 +18,20 @@ enum {
     FILE_MENU_CONFIRM       = 1, // confirmation prompt is open
     FILE_MENU_MESSAGE       = 2, // message is displayed, e.g. "File X has been deleted."
     FILE_MENU_INPUT_NAME    = 3, // "Enter a file name!" screen
-    FILE_MENU_LANGUAGES     = 4,
 };
 
 enum {
     FM_MAIN_SELECT_FILE         = 0, // choose which file to load
     FM_MAIN_SELECT_DELETE       = 1, // choose which file to delete
-    #if !VERSION_PAL
-    FM_MAIN_SELECT_LANGUAGE     = 2,
```

Comment:
Yes, though the asset stack. Apologies, I thought we've discussed that a couple times.

However it's not actually relevant for the changes in this file. Only non-PAL versions have a 'language select' state here for the filemenu, which is just a dummy state copied from the main 'SELECT_FILE' state without functionality. When proper language select was added for PAL, it was implemented with its own game mode instead of piggybacking off the main menu mode.

Reply:
> Apologies, I thought we've discussed that a couple times.

We did, I just didn't realise it involved dropping the language feature altogether. But sure